### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by petejodo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
-React-portal
+React-body-subtree
 ============
-[![npm version](https://img.shields.io/npm/v/react-portal.svg?style=flat-square)](https://www.npmjs.com/package/react-portal)
-[![npm downloads](https://img.shields.io/npm/dm/react-portal.svg?style=flat-square)](https://www.npmjs.com/package/react-portal)
-[![Build Status](https://travis-ci.org/tajo/react-portal.svg?branch=master)](https://travis-ci.org/tajo/react-portal)
+**Note** This is a fork of [react-portal](github.com/tajo/react-portal) that fits more to our needs. Development and functionality may divert from the original package to address these concerns.
```
Comment (twhid) [inline README.md]: Add the protocol in the URL to react-portal.

**Comment by petejodo** [comment]
fixed

File: README.md [hunk 2]
```diff
@@ -151,26 +147,8 @@ this.refs.myPortal.openPortal()
...
-npm start
-open http://localhost:3000
-```
-
 **Don't forget to run this before every commit:**
 
 ```
 npm test
 ```
-
```
Comment (twhid) [inline README.md:171]: Add this back, something like:

-## Credits
- [react-portal](https://github.com/tajo/react-portal) originally inspired by the talk [React.js Conf 2015 - Hype!, Ryan Florence](https://www.youtube.com/watch?v=z5e7kWSHWTg) and written by Vojtech Miksu 2015, [miksu.cz](https://miksu.cz), [@vmiksu](https://twitter.com/vmiksu)

Comment (petejodo) [reply README.md:171]: Ok will do

**Pull request merged**